useCase/useCase.go

```
@@ -0,0 +1,40 @@
+package useCase
+
+import (
+	"strconv"
+
+	"github.com/Diegoplas/go-bootcamp-deliverable/app"
+	"github.com/Diegoplas/go-bootcamp-deliverable/utils"
+)
+
+func GetPokemonFromCSV(wantedIndex int) utils.PokemonData {
+
+	csvLines := app.CSVReader()
+
+	for _, line := range csvLines {
+
+		pokemonID, _ := strconv.Atoi(line[0])
```

Don't ignore the error this function returns